lang/python38/files/patch-configure.diff

```diff
@@ -13,7 +13,7 @@
    # has no effect, don't bother defining them
    Darwin/[6789].*)
      define_xopen_source=no;;
--  Darwin/1[0-9].*)
+-  Darwin/[12][0-9].*)
 +  Darwin/[1-9][0-9].*)
      define_xopen_source=no;;
    # On AIX 4 and 5.1, mbstate_t is defined only when _XOPEN_SOURCE == 500 but
```

Comment:
Should this hunk be kept, or dropped in favor of upstream's change (see python/cpython#21113)? (I.e. do we really need to worry about Darwin 30 anytime soon, or wait until Darwin 100?)

Reply:
remove the patch and go with the upstream solution

Reply:
Done in c0694eb1034